app/api/items/get_navigation_data.feature

```diff
@@ -290,7 +293,8 @@ Feature: Get item for tree navigation
                   "partial_access": true,
                   "full_access": true,
                   "gray_access": true
-                }
+                },
+                "children": null
```

Comment:
is there a why this "children" is null while the prev one is empty array? (I've checked into details yet)

Reply:
discussed in Slack